opencraft/settings.py

```diff
@@ -295,6 +295,10 @@
 # for provisioning Open edX app servers.
 DEFAULT_CONFIGURATION_VERSION = env('DEFAULT_CONFIGURATION_VERSION', default=DEFAULT_OPENEDX_RELEASE)
 
+# Git tag for latest Open edX release. Used as a default refspec for
+# configuration, edx-platform, forum, notifier, xqueue, and certs when creating production instances.
+LATEST_OPENEDX_RELEASE = env('LATEST_OPENEDX_RELEASE', default='named-release/dogwood.rc')
```

Comment:
@antoviaque I would suggest to use our own branch name, like `opencraft/dogwood` for the Dogwood release we use on the our edx-platform fork, since we want to add custom commits to it.  I would build on the latest release tag `dogwood.3` and add whatever additional commits we need on top of that.  Same thing for the configuration repo.